charts/consul/test/unit/webhook-cert-manager-podsecuritypolicy.bats

```diff
@@ -7,6 +7,16 @@ load _helpers
   assert_empty helm template \
       -s templates/webhook-cert-manager-podsecuritypolicy.yaml  \
       --set 'connectInject.enabled=false' \
+      --set 'global.enablePodSecurityPolicies=true' \
```

Comment:
Shouldn't this be disabled as per the test case?

Reply:
It should be `connect:false` and `psp: true`, we're testing if turning off connect makes the PSP not template out.